analysis/coverage_data_utils.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for coverage data calculation."""
+
+import posixpath
```

Comment:
nit: according to the styleguide imports should be in alphabetical order (without respecting case, see https://google.github.io/styleguide/pyguide.html#313-imports-formatting). So this order should be:
```
import collections
import json
import tempfile
import os
import pandas as pd
import tempfile

from common import filestore_utils
```